lib/facter/simplib_sysctl.rb

```diff
@@ -0,0 +1,46 @@
+# _Description_
```

Comment:
This *is* the better way.

The *other* way is to dump all of sysctl into a hash, but I don't think we want that madness in every `facter` run.